build.sbt

```diff
@@ -52,44 +52,44 @@ ThisBuild / scalaVersion := Scala213
 ThisBuild / githubWorkflowPublishTargetBranches := Seq() // disable publication for now
 
 ThisBuild / githubWorkflowBuildMatrixAdditions +=
-  "platform" -> List("jvm", "js")
+  "platform" -> List("jvm", "js", "native")
 
 ThisBuild / githubWorkflowBuildMatrixExclusions ++=
-  githubWorkflowJavaVersions.value.filterNot(Set(PrimaryJava)).map { java =>
-    MatrixExclude(Map("platform" -> "js", "java" -> java))
+  githubWorkflowJavaVersions.value.filterNot(Set(PrimaryJava)).flatMap { java =>
+    Seq(MatrixExclude(Map("platform" -> "js", "java" -> java)),
+        MatrixExclude(Map("platform" -> "native", "java" -> java))
+    )
+  }
+
+ThisBuild / githubWorkflowBuildMatrixExclusions ++=
+  Seq("jvm", "js", "native").map { platform =>
+    MatrixExclude(
+      Map("platform" -> platform, "java" -> LatestJava, "scala" -> DottyOld)
```

Comment:
You can rebase on latest master, this is not necessary anymore.

Reply:
Done.